### PR TITLE
doc: Update the information provided by pilot users 

### PR DESCRIPTION
**Pull request by Thealean28**
# Description
Changes requested by pilot users in S3 have been added to the change log. Also, a new column has been added to inform about the change progress.

# Checklist:
- [x] I have performed a self-review of my code running `npm run build` and `npm run serve` after pushing the code.
- [x] I have checked that there is no fragile information published unintentionally.

# Screenshot:
image

### CODE REVIEW

File: docs/04. Seguimiento/4.8. Registro de cambios.mdx [hunk 1]
```diff
@@ -24,29 +25,40 @@ planificación inicial.
...
+| 9  | Validación de los campos fecha de nacimiento, dirección, teléfono y números expediente en el formulario para actualizar un beneficiario | <span style={{backgroundColor: 'red', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Alta</span>     | 11 de abril de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'orange'}}>En progreso</span>|
+| 10  | Poder actualizar el género de los miembros de una familia |   <span style={{backgroundColor: 'orange', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Media</span>    | 11 de abril de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'orange'}}>En progreso</span>|
```
Comment (AntonioRodriguezRuiz) [inline docs/04. Seguimiento/4.8. Registro de cambios.mdx:39]: Number 10 is complete

File: docs/04. Seguimiento/4.8. Registro de cambios.mdx [hunk 2]
```diff
@@ -24,29 +25,40 @@ planificación inicial.
...
+| 8  | Incluir los campos número de citas totales, edad y si el servicio que tenían que proporcionarles está finalizado en el listado de beneficiarios | <span style={{backgroundColor: 'orange', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Media</span>    | 11 de abril de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'orange'}}>En progreso</span>|
+| 9  | Validación de los campos fecha de nacimiento, dirección, teléfono y números expediente en el formulario para actualizar un beneficiario | <span style={{backgroundColor: 'red', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Alta</span>     | 11 de abril de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'orange'}}>En progreso</span>|
```
Comment (AntonioRodriguezRuiz) [inline docs/04. Seguimiento/4.8. Registro de cambios.mdx:38]: Number 9 is complete

File: docs/04. Seguimiento/4.8. Registro de cambios.mdx [hunk 3]
```diff
@@ -24,29 +25,40 @@ planificación inicial.
...
+| 7  | Mostrar mensajes de error en el formulario de login | <span style={{backgroundColor: 'orange', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Media</span>     | 11 de abril de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'orange'}}>En progreso</span>|
+| 8  | Incluir los campos número de citas totales, edad y si el servicio que tenían que proporcionarles está finalizado en el listado de beneficiarios | <span style={{backgroundColor: 'orange', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Media</span>    | 11 de abril de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'orange'}}>En progreso</span>|
```
Comment (AntonioRodriguezRuiz) [inline docs/04. Seguimiento/4.8. Registro de cambios.mdx:37]: This is already covered by the list of interventions on the right panel of a beneficiary detail page and the option to "Finalize" someone

File: docs/04. Seguimiento/4.8. Registro de cambios.mdx [hunk 4]
```diff
@@ -24,29 +25,40 @@ planificación inicial.
...
+| 6  | Mostrar estadísticas requeridas por el banco de alimentos | <span style={{backgroundColor: 'red', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Alta</span>     | 11 de abril de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'orange'}}>En progreso</span>|
+| 7  | Mostrar mensajes de error en el formulario de login | <span style={{backgroundColor: 'orange', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Media</span>     | 11 de abril de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'orange'}}>En progreso</span>|
```
Comment (AntonioRodriguezRuiz) [inline docs/04. Seguimiento/4.8. Registro de cambios.mdx:36]: Number 7 is complete

File: docs/04. Seguimiento/4.8. Registro de cambios.mdx [hunk 5]
```diff
@@ -24,29 +25,40 @@ planificación inicial.
...
+| 18 | Fecha de intervencion en formato: fecha y hora | <span style={{backgroundColor: 'red', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Alta</span>   | 22 de marzo de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'green'}}>Realizado</span>|
+| 19 | Área desplegable que contenga: Prevención, Atención, Incorporación sociolaboral, Otro (Observaciones) | <span style={{backgroundColor: 'red', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Alta</span>| 22 de marzo de 2024 | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'orange'}}>En progreso</span>|
```
Comment (AntonioRodriguezRuiz) [inline docs/04. Seguimiento/4.8. Registro de cambios.mdx:48]: Number 19 is complete

File: docs/04. Seguimiento/4.8. Registro de cambios.mdx [hunk 6]
```diff
@@ -24,29 +25,40 @@ planificación inicial.
...
+| 20 | Alias por defecto: 2 primeras letras del primer apellido y 2 del segundo. Si el nombre  es simple 2 primeras letras del nombre. Si es compuesto primera letra de cada nombre. A veces los alias son personalizados | <span style={{backgroundColor: 'red', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Alta</span> | 22 de marzo de 2024 | Incluido en la planificacion      | Sí                           | Sí                             |<span style={{color: 'green'}}>Realizado</span>|
+| 21 | Poder filtrar por fechas en las intervenciones | <span style={{backgroundColor: 'orange', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Media</span>    | 22 de marzo de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'orange'}}>En progreso</span>|
```
Comment (AntonioRodriguezRuiz) [inline docs/04. Seguimiento/4.8. Registro de cambios.mdx:50]: Number 21 is in review (complete)

File: docs/04. Seguimiento/4.8. Registro de cambios.mdx [hunk 7]
```diff
@@ -24,29 +25,40 @@ planificación inicial.
...
+| 26 | Mostrar mensajes de avisos en la aplicación | <span style={{backgroundColor: 'orange', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Media</span>   | 18 de marzo de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'green'}}>Realizado</span>|
+| 27 | Notificar cuando queda un tiempo determinado | <span style={{backgroundColor: 'orange', display: 'inline-block', width: '100%', height: '100%', textAlign: 'center'}}>Media</span>   | 18 de marzo de 2024         | Incluido en la planificacion                                                           | Sí                           | Sí                             |<span style={{color: 'orange'}}>En progreso</span>|
```
Comment (AntonioRodriguezRuiz) [inline docs/04. Seguimiento/4.8. Registro de cambios.mdx:56]: Number 27 has been affected by a change in the scope of the project

**Review by AntonioRodriguezRuiz — CHANGES_REQUESTED** [review]

File: docs/04. Seguimiento/4.8. Registro de cambios.mdx (hunk 1 above)
Comment (Thealean28) [reply docs/04. Seguimiento/4.8. Registro de cambios.mdx]: Done!

File: docs/04. Seguimiento/4.8. Registro de cambios.mdx (hunk 2 above)
Comment (Thealean28) [reply docs/04. Seguimiento/4.8. Registro de cambios.mdx]: Done!

File: docs/04. Seguimiento/4.8. Registro de cambios.mdx (hunk 3 above)
Comment (Thealean28) [reply docs/04. Seguimiento/4.8. Registro de cambios.mdx]: I deleted it

File: docs/04. Seguimiento/4.8. Registro de cambios.mdx (hunk 4 above)
Comment (Thealean28) [reply docs/04. Seguimiento/4.8. Registro de cambios.mdx]: Done!

File: docs/04. Seguimiento/4.8. Registro de cambios.mdx (hunk 5 above)
Comment (Thealean28) [reply docs/04. Seguimiento/4.8. Registro de cambios.mdx]: Done!

File: docs/04. Seguimiento/4.8. Registro de cambios.mdx (hunk 6 above)
Comment (Thealean28) [reply docs/04. Seguimiento/4.8. Registro de cambios.mdx]: Done!

File: docs/04. Seguimiento/4.8. Registro de cambios.mdx (hunk 7 above)
Comment (Thealean28) [reply docs/04. Seguimiento/4.8. Registro de cambios.mdx]: I updated the state to "cancelled"

**Pull request merged**